Tests/SwiftSyntaxBuilderTest/DictionaryExprTests.swift

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class DictionaryExprTests: XCTestCase {
+  func testPlainDictionaryExpr() {
+    let buildable = DictionaryExpr {
+      for i in 1...3 {
+        DictionaryElement(keyExpression: IntegerLiteralExpr(i), valueExpression: IntegerLiteralExpr(i))
+      }
+    }
+    AssertBuildResult(buildable, "[1: 1, 2: 2, 3: 3]")
+  }
+
+  func testEmptyDictionaryExpr() {
+    let buildable = DictionaryExpr()
+    AssertBuildResult(buildable, "[:]")
+  }
+
+  func testMultilineDictionaryLiteral() {
+    let builder = DictionaryExpr("""
+      [
+        1:1,
+      2: "2⃣️",
```

Comment:
Could you not use `2⃣️`? We use these number emojis in the parser tests to mark diagnostic locations in the source file and this just starts my brain wondering what the `2⃣️` marks here. Any other emoji is fine.

Reply:
Didn’t have known that! Replaced with some CJK fancy:)